interactions.proto

```diff
@@ -103,3 +103,15 @@ message Location {
   double latitude = 1;
   double longitude = 2;
 }
+
+message RequestValidationResult {
```

Comment:
It would be good to comment on a) who sends, receives  b) what is the purpose/context of these messages.

Reply:
Sure, I can adjust the interaction diagrams. These would only appear in a misformed request.